go/internal/feast/registry/registry.go

```diff
@@ -36,6 +37,17 @@ type Registry struct {
 	mu                             sync.Mutex
 }
 
+func (r *Registry) CacheFeatureService(featureService *core.FeatureService) {
```

Comment:
Please remove this function, since it's not used anymore

Reply:
Fixed.